domains/certificates/Query.API/API/MeasurementsSyncer/MeasurementsSyncService.cs

```diff
@@ -3,67 +3,58 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+using API.Configurations;
 using API.MeasurementsSyncer.Metrics;
 using API.MeasurementsSyncer.Persistence;
 using DataContext.Models;
 using DataContext.ValueObjects;
 using Measurements.V1;
 using Meteringpoint.V1;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
 
 namespace API.MeasurementsSyncer;
 
-public class MeasurementsSyncService
+public class MeasurementsSyncService(
+    ILogger<MeasurementsSyncService> logger,
+    ISlidingWindowState slidingWindowState,
+    Measurements.V1.Measurements.MeasurementsClient measurementsClient,
+    SlidingWindowService slidingWindowService,
+    IMeasurementSyncMetrics measurementSyncMetrics,
+    IMeasurementSyncPublisher measurementSyncPublisher,
+    Meteringpoint.V1.Meteringpoint.MeteringpointClient meteringPointsClient,
+    IOptions<MeasurementsSyncOptions> options)
```

Comment:
Why are you removing the underscores? Martin is adding them

Reply:
Because they are no longer private variables. It is using primary constructors now:

[https://devblogs.microsoft.com/dotnet/csharp-primary-constructors-refactoring/](https://devblogs.microsoft.com/dotnet/csharp-primary-constructors-refactoring/)